### ISSUE TITLE
Airplane does not land. It raises the nose on final

### ISSUE THREAD

**Issue by renechoppa** [flybywiresim/a32nx#4920]
### Aircraft Version

Stable

### Build info

```json
{
  "built": "2021-05-05T23:03:06+00:00",
  "ref": "v0.6.1",
  "sha": "4a1cccc90ab3997109e394402ab9b41a80fbf325",
  "actor": "Benjozork",
  "event_name": "manual"
}
```

### Describe the bug

Airplane on final, raises the nose inadvertently, no matter what input I try. AP disconnected or not, AT disconnect or not. All versions.

### Expected behavior

I simply expect to proceed on final normally, either using the AP or mannually. 

### Steps to reproduce

1. The whole flight the airplane seems to behave normally maybe because the AP is on, even though on take off the attitude seems a bit weird.
2. When I'm about to start configuring the flaps the beheviour of the airplane changes. It sometimes capture the final approach (ILS or RNAV) and sometimes it doesn't. 
3. But the real problem begins when I'm approaching the terrain. At about 1500 feet it raises the nose as if I was going around. If I don't add power, it eventually stalls the airplane.

### Discord Username (optional)

Glider171#4575

**Comment by aguther** [comment]
Which flight model do you use in MSFS settings? Is it the legacy?

**Comment by aguther** [comment]
Legacy flight model is not supported, please switch to Modern.

**Comment by renechoppa** [comment]
Yes it solved the problem. It seems a bit artificial and I had difficulty to lose speed on final aproach, even with the throttle in idle but I've noticed one good aspect: the airplane does not swing its wings anymore on takeoff and landing. Now, comes the question: Am I the only one that didn't know about flight model? If the answer is yes, than shame on me. If not, then I should be well stated somewhere like the main page of the downloader. But tks anyway. I will proceed on new test here. Have a good one Adreas.

**Comment by aguther** [comment]
Great that it's solved for you. It's at least stated on the page for the development version. My feeling is that whatever we put it some people will not catch it. But there are good news: we just discovered that with SU4 there is now a new parameter for the flight model that can force to modern flight model independently of the user setting. This will eventually be added then we should be fine.

**Comment by renechoppa** [comment]
Great to hear about it. However, now there is one more problem. Using an old version, a couple of weeks ago, I had an issue with FMS. When I selected the Flight Plan button and chose runway and SID, the FMS freezes, not allowing me to INSERT it. Other pages like FERF still work but from that point on, many things stop working. I thought it was a conflict because it happened specifically in one airport (defaut) but I installed photogrammetry of the area around. But yesterday, it happened again in a different airport. I'm using the stable version as mentioned above: {
  "built": "2021-05-05T23:03:06+00:00",
  "ref": "v0.6.1",
  "sha": "4a1cccc90ab3997109e394402ab9b41a80fbf325",
  "actor": "Benjozork",
  "event_name": "manual"
}
Any idea?
